components/admin_console/admin_settings.tsx

```diff
@@ -55,6 +55,7 @@ export default abstract class AdminSettings extends React.PureComponent<Props, S
 
     protected abstract getStateFromConfig(config: AdminConfig): State;
 
+    // eslint-disable-next-line @typescript-eslint/ban-types
```

Comment:
I assume the `object` type is banned by default? Would it make sense to use `any` instead?

Reply:
yes, `object` is banned. I think you're right,`any`, or maybe `Record<string, any>` would be better than an ignore comment.

Reply:
Either should be fine here. The whole way that AdminSettings uses inheritance is already an anti-pattern for React (my bad 😛), so if we have to add some special handling to get around it, I'm fine with that.